src/main/python/prunetags/api.py

```diff
@@ -222,6 +222,63 @@ def remove_prerelease_tags(self, repo=None, branch=None, noop=True):
                 logger.error(f'error occurred removing tag {prerelease_tag} from repo {repo}: {exception}')
         logger.debug(f'removed prerelease tags from repo {repo}')
 
+    def get_version_tags(self, repo=None, branch=None, expression=None):
+        """ return version tags, latest version and latest version sha for repo
+        """
+        logger.info(f'getting version tags for repo {repo}')
+        tags = self.read_all(f'/repos/{repo}/tags')
+        if not tags:
+            logger.info(f'repo {repo} has no tags')
+            return
+        latest_version, latest_version_sha = self.get_latest_version(repo=repo, tags=tags, branch=branch)
+        if not latest_version:
+            logger.info(f'repo {repo} has no tags that are versions')
+            return
+        logger.info(f'repo {repo} latest tag version is {latest_version}')
+        exclude = None
+        if latest_version.prerelease != ():
+            exclude = latest_version
+        version_tags = GitHubAPI.filter_version_tags(tags=tags, exclude=exclude, expression=expression)
+        logger.info(f'repo {repo} has {str(len(version_tags)).zfill(3)} version tags set for removal according to expression="{expression}"')
```

Comment:
update the log string and remove `for removal` since this method is about getting version tags it shouldn't know about removing them